uitest/Pages/UtilitiesPage.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Security.Principal;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Win32.SafeHandles;
+using OpenQA.Selenium.Appium.Windows;
+using Windows.Win32;
+
+namespace DevHome.UITest.Pages;
+
+public class UtilitiesPage : ApplicationPage
+{
+    public UtilitiesPage(WindowsDriver<WindowsElement> driver)
+        : base(driver)
+    {
+    }
+
+    public void LaunchAndVerifyUtility(string utilityName, bool launchAsAdmin = false)
+    {
+        Trace.WriteLine($"Launching {utilityName} with admin: {launchAsAdmin}");
+        WindowsElement utilityView = Driver.FindElementByAccessibilityId(utilityName);
+
+        if (launchAsAdmin)
+        {
+            var launchAsAdminToggle = utilityView.FindElementByAccessibilityId("AdminToggleAutomationId");
+            launchAsAdminToggle.Click();
+        }
+
+        var launchButton = utilityView.FindElementByAccessibilityId("LaunchButtonAutomationId");
+        launchButton.Click();
+
+        // Wait for the utility to launch
+        Thread.Sleep(2000);

```

Comment:
any possibility to make this event driven ie some sort of process watcher? possibly using WMI?